.github/workflows/check.yml

```diff
@@ -86,6 +86,7 @@ jobs:
         tox_env:
           - dev
           - pkg_check
+          - mypy
```

Comment:
let's call it type instead of mypy; it's a type check, mypy is an implementation detail